schemas/answers/definitions.json

```diff
@@ -11,7 +11,7 @@
       "type": "object",
       "properties": {
         "label": {
-          "type": "string"
+          "$ref": "../string_interpolation/definitions.json#/string_with_placeholders"
```

Comment:
Not able to use `string_with_placeholders` on option labels because it errors on existing check that doesn't know about the new possible structure of label. So needs some more code changes sry 👀 
```
 File "/Users/abrahm/Documents/projects/eq-schema-validator/app/validation/validator.py", line 115, in _validate_questions
    errors.extend(self.validate_duplicate_options(answer))
  File "/Users/abrahm/Documents/projects/eq-schema-validator/app/validation/validator.py", line 585, in validate_duplicate_options
    if option['label'] in labels:
TypeError: unhashable type: 'dict'
```

Also some minor linting issues:
```
************* Module app.validation.validator
app/validation/validator.py:804:0: C4001: Invalid string quote ", should be ' (invalid-string-quote)
app/validation/validator.py:814:0: C4001: Invalid string quote ", should be ' (invalid-string-quote)
```

Reply:
I've added some test schema for placeholders in radio and checkbox option labels.